.travis.yml

```diff
@@ -25,10 +27,18 @@ install:
 
 before_script:
   # Install Varnish
-  - curl http://repo.varnish-cache.org/debian/GPG-key.txt | sudo apt-key add -
-  - echo "deb http://repo.varnish-cache.org/ubuntu/ precise varnish-${VARNISH_VERSION}" | sudo tee -a /etc/apt/sources.list
-  - sudo apt-get update -qq
-  - sudo apt-get install -qq varnish python-sphinx enchant
+  - |
+    if [ "$VARNISH_VERSION" = "5.0" ]; then
+      wget https://repo.varnish-cache.org/pkg/5.0.0/varnish_${VARNISH_VERSION}.0-1_amd64.deb
+      sudo dpkg -i varnish_${VARNISH_VERSION}.0-1_amd64.deb
+      sudo apt-get install -f libpcre3
+    else
+      curl http://repo.varnish-cache.org/debian/GPG-key.txt | sudo apt-key add -
+      echo "deb http://repo.varnish-cache.org/ubuntu/ precise varnish-${VARNISH_VERSION}" | sudo tee -a /etc/apt/sources.list
+      sudo apt-get update -qq
+      sudo apt-get install -qq varnish
+    fi
+  - sudo apt-get update -qq && apt-get install -qq python-sphinx enchant
```

Comment:
you lost the other sudo here. maybe just make that two lines again, no need to chain the two things with &&.

or move the update into the varnish 5 case